content/tracing/advanced_usage/_index.md

```diff
@@ -472,6 +490,30 @@ using(var scope = Tracer.Instance.StartActive("web.request"))
 
 [dotnet compatibility]: /tracing/setup/dotnet/#compatibility
 
+{{% /tab %}}
+{{% tab "PHP" %}}
+If you aren’t using libraries supported by automatic instrumentation (see [library compatibility][php compatibility]), manually instrument your code.
+
+The following example uses the global Datadog Tracer and creates a span to trace a web request:
+
+```php
+use OpenTracing\GlobalTracer;
+use DDTrace\Tags;
```

Comment:
```suggestion
use DDTrace\Tags;
use DDTrace\Types;
```
[not a blocker] I am not sure this is required as we declare the files in the `composer.json` but if we add `Tags` then we should also add `Types`.